app/AndroidManifest.xml

```diff
@@ -3,7 +3,7 @@
           xmlns:tools="http://schemas.android.com/tools"
           package="org.commcare.dalvik"
           android:versionCode="106"
-          android:versionName="2.49">
+          android:versionName="2.48.0">
```

Comment:
@ShivamPokhriyal Flagging that this should have been 2.49 (master points to the next major version)